atom/common/api/atom_bindings.h

```diff
@@ -11,6 +11,9 @@
 #include "base/strings/string16.h"
 #include "v8/include/v8.h"
 #include "vendor/node/deps/uv/include/uv.h"
+#include "atom/common/crash_reporter/crash_reporter.h"
+
+using crash_reporter::CrashReporter;
```

Comment:
Turns out I didn't really need crash_reporter.h here. Can make do with simple maps.
Also ran lint-cpp and fixed all linting issues